obspy/io/nordic/core.py

```diff
@@ -77,13 +77,16 @@ def _is_sfile(sfile, encoding='latin-1'):
         except Exception:
             return False
     else:
-        tags = _get_line_tags(f=sfile)
+        try:
+            tags = _get_line_tags(f=sfile)
+        except Exception:
+            return False
     if tags is not None:
         # Note that there can be two origin lines, but only to allow more
         # magnitudes, not more origins.
         try:
             head_line = tags['1'][0][0]
-        except KeyError:
+        except IndexError:
```

Comment:
without knowing details.. there's three nested `get` operations in one single try/except.. looks fragile

Reply:
Thanks for having a look @megies, but I'm sorry, I don't get this comment, could you explain a bit more (sorry!).